resources/views/home.blade.php

```diff
@@ -1,9 +1,25 @@
 @extends('layouts.app')
 
 @section('content')
-<div class="container">
+
+<style>
+.display-5 {
+    font-family: 'Marcellus', serif;
+}
+
+.p1 {
+    font-family: 'Marcellus', serif;
+}
+
```

Comment:
Have a separate file for this css.

Reply:
I have moved the css regarding fonts to a separate file.